GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -306,8 +306,9 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(RevisionGridControl.Command.ResetRevisionPathFilter, Keys.Control | Keys.Shift | Keys.H),
                     Hk(RevisionGridControl.Command.ResetRevisionFilter, Keys.Control | Keys.Shift | Keys.I),
                     Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.I),
-                    Hk(RevisionGridControl.Command.SelectCurrentRevision, Keys.Control | Keys.Shift | Keys.C),
                     Hk(RevisionGridControl.Command.SelectAsBaseToCompare, Keys.Control | Keys.L),
+                    Hk(RevisionGridControl.Command.SelectCurrentRevision, Keys.Control | Keys.Shift | Keys.C),
+                    Hk(RevisionGridControl.Command.SelectFirstDiffGroupChanges, Keys.Control | Keys.A),
```

Comment:
Why is this needed as a new hotkey?

Reply:
> Why is this needed as a new hotkey?

Because of the the case where the RevisionGrid is focused:
`GitExtensionsControl.ProcessCmdKey` does not find a Hotkey for the RevisionGrid and calls the base method `UserControl.ProcessCmdKey`.
This gives `DataGridView` the chance to handle the key (and to select all revisions) before `FormBrowse.ProcessCmdKey` could route it to `revisionDiff`.
But `FormBrowse.ProcessCmdKey` would not route it anyway, because `IsTextEditKey` returns `true` for `Ctrl+A` since this PR.
(Adding explicit Hotkeys for `Ctrl+A`, `Ctrl+C`, `Ctrl+V` to all controls is not an option IMO.)

I have updated the PR description text.